### DISCUSSION TITLE
posts/how-to-create-your-own-website/

### DISCUSSION THREAD

**Discussion by giscus** [michaelneuper/website#42] [General]
# posts/how-to-create-your-own-website/

Setting up your own static website can be a great way to showcase your personal brand, portfolio, or business online. In this post, we’ll walk through the process of setting up a static website using Hugo as the static site generator and Netlify as the hosting service.
Building The Site First, let’s talk about what a static website is and why it’s a good choice for many people. A static website is a website that is built using only HTML, CSS, and JavaScript files.

https://michaelneuper.com/posts/how-to-create-your-own-website/

**Comment by davidzhquan** [comment]
uggi
